scripts/portfolio_view.js

```diff
@@ -13,3 +13,4 @@ portfolioView.handleMainNav = function () {
 };
 
 portfolioView.handleMainNav();
```

Comment:
the handleMainNav function hides the tab-content, but it doesn’t show the correct content. In the index.html page, the list item for the About page needs a data-content attribute labeled after its respective section’s id which would be "about" here.